Stress analyzer/ViewModels/SurveyResponseViewModel.swift

```diff
@@ -61,10 +61,13 @@ final class SurveyResponseViewModel: ObservableObject {
             let (data, _) = try await URLSession.shared.data(for: request)
             surveyResp = try JSONDecoder().decode([SurveyResponse].self, from: data)
         
-            print(surveyResp)
+            print(surveyResp[1])
```

Comment:
@akhor005, is there a reason to print a specific index rather than the list of returned survey responses? if there's only one response returned, this will throw an error and jump to the catch statement and print error when it may actually store responses just fine